undertow/src/test/java/org/wildfly/extension/undertow/ServerServiceTestCase.java

```diff
@@ -0,0 +1,195 @@
+/*
```

Comment:
@fl4via I don't understand how this test is relevant in EAP7-1187

Reply:
This is a unit test covering the new logic in org.wildfly.extension.undertow.Server, which is the core logic of EAP7-1187.